src/main/java/net/sf/jabref/logic/formatter/Formatters.java

```diff
@@ -45,7 +45,7 @@
 public class Formatters {
 
     public static final List<Formatter> CONVERTERS = Arrays.asList(
-            new HtmlToLatexFormatter(),
+            new HtmlToLatexFormatter(false), // TODO: Get the correct value from USE_CONVERT_TO_EQUATION
```

Comment:
Maybe this isn't needed and we just can decide on one way to convert subscripts and superscripts from HTML to LaTeX. It is there from historical reasons, where I first did it using equations and then someone wanted the `\textsuperscript` version. I would almost suggest always using `\textsuperscript` now that the table view can render it properly (earlier it was more compact with `$^2$` compard to `\textsuperscript{2}`.

This would also reduce the amount of settings.